src/App.js

```diff
@@ -1,26 +1,27 @@
-import React, {useState} from "react";
-import Calendar from 'react-calendar'
+import React, { useState } from "react";
+import Calendar from "react-calendar";
 
 function App() {
-    const [date, setDate] = useState(new Date())
-    const onChange = date => {
-        setDate(date)
-    }
+  const [date, setDate] = useState(new Date());
+  const onChange = (date) => {
+    setDate(date);
+  };
 
-    return (
-        <div className={"w-full"}>
-            <div className={"m-auto flex flex-col justify-center text-center max-w-[300px]"}>
-                <Calendar
-                    className={"border-2"}
-                    onChange={onChange}
-                    value={date}
-                />
-                <div className={"text-center py-7 text-red-700"}>
-                    {date.toDateString()}
-                </div>
-            </div>
+  return (
+    <div className={"w-full"}>
+      <div
+        className={
+          "m-auto flex flex-col justify-center text-center max-w-[300px]"
+        }
+      >
+        <Calendar className={"border-2"} onChange={onChange} value={date} />
+        {/* one format here */}
```

Comment:
It should be choosen same language format here. English or russian
![Screenshot 2024-01-22 at 18 48 51](https://github.com/ViktorProjects/calendar_main/assets/106515539/7009e7de-0ca2-4410-9b90-6b5c3bd84309)